cliquet/__init__.py

```diff
@@ -26,7 +26,7 @@
 DEFAULT_SETTINGS = {
     'cliquet.backoff': None,
     'cliquet.batch_max_requests': 25,
-    'cliquet.cache_backend': 'cliquet.cache.redis',
+    'cliquet.cache_backend': 'cliquet.cache.memory',
```

Comment:
This PR prevents overriding the user settings (in her .ini file) with Kinto default settings.

What this PR doesn't do (and maybe should do) is to let kinto override cliquet default settings.